subsys/net/ip/net_context.c

```diff
@@ -1496,6 +1513,32 @@ static int get_context_reuseport(struct net_context *context,
 #endif
 }
 
+static int get_context_ipv6_v6only(struct net_context *context,
+				   void *value, size_t *len)
+{
+#if defined(CONFIG_NET_IPV4_MAPPING_TO_IPV6)
+	if (!value || !len) {
+		return -EINVAL;
+	}
+
+	if (*len != sizeof(int)) {
+		return -EINVAL;
+	}
+
+	if (context->options.ipv6_v6only == true) {
+		*((int *)value) = (int) true;
+	} else {
+		*((int *)value) = (int) false;
+	}
```

Comment:
```suggestion
	*((int *)value) = (int)context->options.ipv6_v6only;
```

Reply:
I decided not to do this in order this function look similar as the other get socket option functions.